s2i/bin/assemble

```diff
@@ -46,7 +46,7 @@ fi
 
 if [[ -f requirements.txt ]]; then
   echo "---> Installing dependencies ..."
-  pip install -r requirements.txt
+  pip install -r requirements.txt --no-deps
```

Comment:
`pip install --no-deps -r requirements.txt`. Please add also this in line 54:
```
pip install --no-deps .
```

Reply:
Ok. Done